crates/starknet-server/src/cli.rs

```diff
@@ -43,6 +79,16 @@ impl Args {
                 .clone()
                 .try_into()
                 .expect("Invalid value for initial balance"), // TODO doesn't exit nicely
+            host: self.host.clone(),
+            port: self.port,
+            timeout: self.timeout,
+            gas_price: self.gas_price,
+            chain_id: match self.chain_id.as_str() {
+                "MAINNET" => StarknetChainId::MainNet,
+                "TESTNET" => StarknetChainId::TestNet,
+                "TESTNET2" => StarknetChainId::TestNet2,
+                _ => StarknetChainId::TestNet,
```

Comment:
This is wrong, it means if I do `--chain-id lmao-i-am-an-invalid-chain-id`, it will default to TESTNET.

Reply:
I thought that it's expected behavior but I'll raise exception in that case